<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>30</pointsize>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;
border:2px solid white;</string>
    </property>
    <property name="text">
     <string>Select A Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="ballImage">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>ballImage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="ColorBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="redColor">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>red</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="yellowColor">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>yellow</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="greenColor">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>green</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="blueColor">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>blue</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="purpleColor">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>purple</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pinkColor">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>pink</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="coordinateLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
size:10px</string>
    </property>
    <property name="text">
     <string>coordinate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:white;
border:none</string>
    </property>
    <property name="text">
     <string>Now Start Your Game!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GameInfoBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="title">
     <string>Game Info:</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumberEnemy">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>281</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumberYour">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="EnemysTimerLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Enemy's Timer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="YourTimerLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>15</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Your Timer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="MyBall">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MyBall</string>
     </property>
    </widget>
    <widget class="QLabel" name="EnemyBall">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>EnemyBall</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="loadLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>Loading...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>111</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LeftBottomLayout">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(90, 183, 180);
color:rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scoreButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(90, 183, 180);
color:rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Score</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(90, 183, 180);
color:rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionCreate_the_connection"/>
    <addaction name="actionConnect_to_Server"/>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionAdmit_defeat"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_the_connection">
   <property name="text">
    <string>Create the connection</string>
   </property>
  </action>
  <action name="actionConnect_to_Server">
   <property name="text">
    <string>Connect to Server</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionAdmit_defeat">
   <property name="text">
    <string>Admit defeat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectButton</sender>
   <signal>clicked()</signal>
   <receiver>ColorBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
